crates/tests/win32/tests/winsock.rs

```diff
@@ -0,0 +1,65 @@
+use windows::Win32::Networking::WinSock::*;
+
+#[test]
+fn in_addr() {
+    let ipv4_addr = std::net::Ipv4Addr::new(4, 3, 2, 1);
+    let in_addr: IN_ADDR = ipv4_addr.into();
+
+    assert_eq!(unsafe { in_addr.S_un.S_un_b.s_b1 }, 1);
```

Comment:
Byte-wise representation has to be the same as seen in IP address above. So s_b1 will be 4, s_b1 will be 3 and so on.

Reply:
We can use [`inet_ntop`](https://microsoft.github.io/windows-docs-rs/doc/windows/Win32/Networking/WinSock/fn.inet_ntop.html) here to get textual representations of IN_ADDR and IN_ADDR6, and then compare them to `.to_string` versions. There may be some issues in ipv6, though. But IN_ADDR6 has no these endianness issues, so this test for ipv6 may be less strict.

Reply:
Done.